onnxruntime/core/providers/qnn/builder/qnn_backend_manager.cc

```diff
@@ -1071,16 +1095,17 @@ Status QnnBackendManager::ExtractProfilingEventExtended(
 #endif
 
   if (!tracelogging_provider_ep_enabled) {
-    if (event_data_extended.version == QNN_PROFILE_DATA_VERSION_1) {
-      outfile << event_data_extended.v1.timestamp << ","
-              << message << ","
-              << ExtractQnnScalarValue(event_data_extended.v1.value) << ","
-              << unit << ","
-              << "BACKEND"
-              << ","
-              << eventLevel << ","
-              << (event_data_extended.v1.identifier ? event_data_extended.v1.identifier : "NULL") << "\n";
-    }
+    // QNN issue, the version number not correct, ticket created
+    // if (event_data_extended.version == QNN_PROFILE_DATA_VERSION_1) {
+    outfile << event_data_extended.v1.timestamp << ","
+            << message << ","
+            << ExtractQnnScalarValue(event_data_extended.v1.value) << ","
```

Comment:
without the version check might this crash?

Reply:
It works, since there's only v1 available so far. line 1090 use it this way already.
Otherwise, the output is empty since the version is not correct. I already reported an issue to QC.

Reply:
Are you certain? From what I recall in debugging this older chipsets (say 8280) went down 1 path, and other chipsets (8380) went down the v1 path. I am not sure removing the version check altogether is a good idea, but could be wrong...

Reply:
I think that was with ETW enabled. see code line 1111 below, for ETW case, LogQnnProfileEventAsTraceLogging doesn't check the version.

Reply:
ok, refreshing my memory and taking into account your comment. I think you are saying that we are in ExtractProfilingEventExtended() because QNN_PROPERTY_PROFILE_SUPPORTS_EXTENDED_EVENT. Then there the version v1 is broken (for now) so it's ok to go ahead and skip the check.

ideally, in ETW we should ALSO check the version, but I think you are saying it's ok for now because we previously didn't check the version in the ETW case